### DISCUSSION TITLE
yarn is no longer updated

### DISCUSSION THREAD

**Discussion by mihail727** [yarnpkg/berry#4205] [Q&A] [answered]
today after updating my system (ArchLinux), I can't run the command `yarn upgrade --latest`
error message: `info There appears to be trouble with your network connection. Retrying...`

`yarn -v`: 1.22.17
`node -v`: 17.7.0
`npm -v`: 8.5.3

**Comment by merceyz** [comment, marked as answer]
This repository is not for Yarn v1, see https://github.com/yarnpkg/yarn/issues/8797

**Reply by mihail727** [reply]
> This repository is not for Yarn v1, see [yarnpkg/yarn#8797](https://github.com/yarnpkg/yarn/issues/8797)

thx, my bad
